src/types/gameConsole.ts

```diff
@@ -57,6 +57,12 @@ interface OutputTokens {
   // TWiLightMenu++ Roms go into the /roms subfolder on the 3DS/DSi SD card
   // @see https://github.com/DS-Homebrew/TWiLightMenu/tree/master/7zfile/roms
   twmenu?: string,
+  
+  // Console brand, name and abbrevited name
+  // @see https://wiki.no-intro.org/index.php?title=Systems
+  consoleBrand?: string,
+  consoleName?: string,
+  consoleAbbrName?: string,
```

Comment:
Do you know what No-Intro's intended use of the abbreviation is? This is the first time I've seen it. I see less value in it than in the other tokens, so I'm inclined to not add it.

Reply:
Hi @emmercm,

Thanks of Igir. Glad I can help with littles contributions.

In my opinion it's a simple way to have an human-readable ID. Easy to manipulate and to read, short, without space, without capital letter.

Many of these abbreviations are in the every-day speech (n64, snes, ps2, ...). 

Many projects have their owns lists :
- Redump : http://wiki.redump.org/index.php?title=System_list
- Batocera : https://wiki.batocera.org/systems
- NineSheetMusic : https://www.ninsheetmusic.org/forum/index.php?topic=11423.0
- Video Game Music Preservation Foundation : https://www.vgmpf.com/Wiki/index.php?title=List_of_Platform_Abbreviations_(A-N)
- Romm : https://github.com/rommapp/romm/wiki/Supported-Platforms (not all are abbreviated, but all are ids)
- Es-de : https://gitlab.com/es-de/emulationstation-de/-/blob/master/USERGUIDE.md#supported-game-systems

Or use them without saying it : 
- goodsets : goodn64, goodgg, goodspc, etc.
- adam : roms/GW, roms/PCE, roms/GG
- retrodeck
- onion

Other :
- https://en.wikibooks.org/wiki/History_of_video_games/List_of_Abbreviations

For me the main problem is to choose the goods abbreviations :
- easy way : it to choose to use No-Intro and Redump lists as there's no convention and no rules that you could extract from existing abbreviations. 
- hard way : choose the bests abbreviations in all existing lists.
- weird way : don't care about all that and compute an id consistent with the console brand and name. 

The easy way is for me the best as it reuses existing lists.

We can rename consoleAbbrName to consoleShortName or consoleID.

And it's make sense if it's required for all GameConsole too.